lua/alex/lazy/copilot.lua

```
@@ -1,17 +1,21 @@
 return {
   {
     "github/copilot.vim",
-    event = "InsertEnter",
+    cmd = "Copilot",
+    event = "BufWinEnter",
     init = function()
-      vim.g.copilot_no_tab_map = true
+      vim.g.copilot_no_maps = true
```

Luacheck (1.2.0)

[warning] mutating non-standard global variable 'vim'

(W112)